app/src/hog.c

```diff
@@ -261,6 +328,59 @@ int zmk_hog_send_consumer_report(struct zmk_hid_consumer_report_body *report) {
     return 0;
 };
 
+#if IS_ENABLED(CONFIG_ZMK_MOUSE)
+
+K_MSGQ_DEFINE(zmk_hog_mouse_msgq, sizeof(struct zmk_hid_mouse_report_body),
+              CONFIG_ZMK_BLE_MOUSE_REPORT_QUEUE_SIZE, 4);
+
+void send_mouse_report_callback(struct k_work *work) {
+    struct zmk_hid_mouse_report_body report;
+    while (k_msgq_get(&zmk_hog_mouse_msgq, &report, K_NO_WAIT) == 0) {
+        struct bt_conn *conn = destination_connection();
+        if (conn == NULL) {
+            return;
+        }
+
+        struct bt_gatt_notify_params notify_params = {
+            .attr = &hog_svc.attrs[13],
```

Comment:
This needs to be updated to `.attr = &hog_svc.attrs[16],`. Each `BT_GATT_CHARACTERISTIC` adds two items to the `attrs` array, so the LED report adds 3 items before the mouse items.

Reply:
Does that mean that this change (taken from the upstream PR) is also insufficient, since it only offsets the index for the consumer report by 2 rather than by 3?

https://github.com/moergo-sc/zmk/pull/25/files#diff-b3e2c5329969f421f8e1dc4e06b59b38e157d9555b2ed0557dc55ce6c9f35097R294

Reply:
No, the consumer report upstream was added after the mouse report, so it doesn't affect the index of the mouse report